levelness_node:
    ros__parameters:  
        engineering:
            - "Normal"    
            - "Concerte"
        option:
            - "RoofLevel"
            - "GroundLevel"
        I_R_C: [-0.159174, -0.00532417, -0.987236, -0.986937, -0.0243564, 0.159257, -0.0248935, 0.999689, -0.00137771]
        fileName: '/home/ubuntu/work/files/data/pointcloud.txt'
        extraParam_ground: 4.57
        extraParam_roof: 5.922